samples/bluetooth/rssi_nrf53/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(rssi_nrf53)
```

Comment:
Should we consider to rename this sample to `rssi_compensation` instead? There are other devices which require some form of compensation, see ERRATA-153 and ERRATA-225.

Another option would be to rename the sample to `scanner_rssi` as it demonstrates sampling rssi as a scanner. We could later consider to add samples demonstrating rssi sampling as an advertiser or in connections.

Reply:
I can rename to scanner_rssi and use ifdefs to execute compensation if running on nRF53

Reply:
ok looks like I'll add the compensation for the other devices too. This time I can use the sensor API for this.